### PR TITLE
Issue #450

### PR DESCRIPTION
**Pull request by vivek-2000**
@avinashkranjan 
Script will follow and send msg 
Fixes: #450

### CODE REVIEW

File: Insta-Bot-Follow-SendMsg/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+What is the objective of the Script
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/README.md:1]: Format the readme properly, The headings need to be levels, level 2(##)

File: Insta-Bot-Follow-SendMsg/instabot.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from openpyxl import load_workbook
+from selenium.webdriver.common.action_chains import ActionChains
+
+
+users=['lit_perception']
+
+USERNAME='enter username'
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/instabot.py:14]: get this from user

File: Insta-Bot-Follow-SendMsg/instabot.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import pandas as pd
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from openpyxl import load_workbook
+from selenium.webdriver.common.action_chains import ActionChains
+
+
+users=['lit_perception']
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/instabot.py:12]: get users from user

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Remove the chrome driver, add the instructions in the readme to place latest chrome driver in the directory

**Comment by vivek-2000** [comment]
@kaustubhgupta  updated

File: Insta-Bot-Follow-SendMsg/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Level2
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/README.md:1]: Here replace level 2 with script name

File: Insta-Bot-Follow-SendMsg/instabot.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from openpyxl import load_workbook
+from selenium.webdriver.common.action_chains import ActionChains
+
+browser = webdriver.Chrome(ChromeDriverManager().install())
+time.sleep(10)
+users=[]
+users.append(input("Enter Users Username Whom You Want to Follow and Send Msg "))
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/instabot.py:14]: how this will take multiple users? It will take all the comma-separated users as strings.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Insta-Bot-Follow-SendMsg/instabot.py (hunk 5 above)
Comment (vivek-2000) [reply Insta-Bot-Follow-SendMsg/instabot.py]: @kaustubhgupta updated

**Comment by vivek-2000** [comment]
@kaustubhgupta updated change required

File: Insta-Bot-Follow-SendMsg/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# instabot.py
+What is the use:
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/README.md:2]:
```suggestion
## What is the use:
```

File: Insta-Bot-Follow-SendMsg/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+3.	Then it will ask for the username and password for the account that will be used to follow and send msg.
+
+4.	Then it will automatically go to the  username of the person whom  you want to follow and send msg.
+
+4.	After pressing enter, it will automatically open the Instagram account of the user and  follow and send msg .
+
+5.	Once done, it will Show the Msg.
+
+Note:
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/README.md:17]:
```suggestion
## Note:
```

File: Insta-Bot-Follow-SendMsg/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+If you are using some other web browser, replace ‘chrome’ by the name of that browser
+Modules and tools used:
+Selenium, getpass and time
+
+
+BY Vivek Kumar Patel
```
Comment (kaustubhgupta) [inline Insta-Bot-Follow-SendMsg/README.md:23]:
```suggestion
## By Vivek Kumar Patel
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by vivek-2000** [comment]
@kaustubhgupta Now updated

**Review by kaustubhgupta — APPROVED** [review]

**Comment by vivek-2000** [comment]
@antrikshmisri @santushtisharma10 can u plz review

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**